README.md

```diff
@@ -25,8 +25,14 @@ Contributing
 Contributions are always welcome. For details on how to contribute, check the [CONTRIBUTING](https://github.com/ruflin/Elastica/blob/master/CONTRIBUTING.md) file.
 
 
-Dependencies
+Versions & Dependencies
 ------------
-| Project | Version | Required |
-|---------|---------|----------|
-|[Elasticsearch](https://github.com/elasticsearch/elasticsearch/tree/6.0)|6.0|yes|
+This project tries to follow Elasticsearch in terms of [End of Life](https://www.elastic.co/support/eol) and maintenance since 5.x
+
+| ruflin/elastica                                                                         | ElasticSearch | elasticsearch-php | PHP      |
```

Comment:
Make `elastica` upper case if possible `Elastica`

Reply:
sure.

Reply:
I've updated title to `Elatica`